.github/workflows/database_backup.yml

```diff
@@ -0,0 +1,28 @@
+name: Backup Database to Azure Storage
+concurrency: build_and_deploy_main
+
+on:
+  workflow_dispatch:
+  # TODO: Uncomment after migration
+  # schedule: # 03:00 UTC
+  #   - cron: "0 3 * * *"
+
+jobs:
+  backup:
+    name: Backup AKS Database (production)
+    if: ${{ github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' }}
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
```

Comment:
I'm not sure this is necessary. Can you try without?

Reply:
will give it a go 👍 
this was as per the README: https://github.com/DFE-Digital/github-actions/blob/master/backup-postgres/README.md